### DISCUSSION TITLE
[2013-12-04] Team GUI Demo Day

### DISCUSSION THREAD

**Discussion by farmaazon** [enso-org/enso#8453] [Weekly Demo Day]
* Implemented #8451 - it does not work yet due to some engine issues, but also I miss some functionality from current Language Server API. [Details here](https://github.com/orgs/enso-org/discussions/6832#discussioncomment-7754474)

![image](https://github.com/enso-org/enso/assets/3919101/1c364a7e-2f90-4397-801d-e05927762658)

**Comment by vitvakatu** [comment]
- Addressing review comments in https://github.com/enso-org/enso/pull/8400 
- It allowed to quickly implement https://github.com/enso-org/enso/pull/8455

https://github.com/enso-org/enso/assets/6566674/2437c038-058e-4133-90f2-e7ad160c4f5c

I was sick for part of the week, but I also talked at https://rustcon.ru/ about Enso and our experience with Rust. The recording (in Russian) will be available soon, I will also prepare an English version of the slides.

**Comment by somebody1234** [comment]
Website:
- Add Pipedrive integration to chatbot - now it only needs API details to get up and running

GUI2:
- Addressed reviews and merged: #8389
- Fixed and merged: #8435
- Implemented: #8440
- Fixed: #8445

- Merged: #8361
  - In `develop` there are further improvements (animations, drag handle, drag metadata)

https://github.com/enso-org/enso/assets/4046547/57bacb78-e457-46bd-b3c7-0e603f4d79bc
